CHANGELOG.md

```diff
@@ -37,6 +37,10 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Technical Changes
 - ROX-12967: Re-introduce `rpm` to the main image in order to be able parse installed packages on RHCOS nodes (from Compliance container)
 - ROX-14280: ACS operator default channel changes from `latest` to `stable`. Users of older versions must follow the upgrade procedure in order to preserve ACS data in case of issues with the upgrade.
+- The k8s-istio.zip file inside of scanner-vuln-updates.zip (the file downloaded from https://install.stackrox.io/scanner/scanner-vuln-updates.zip for updating Scanner vulnerabilities in offline-mode)
+  is no longer needed. We will continue to populate it to support older versions of the product, but it will be ignored.
```

Comment:
`We will continue to populate it to support older versions of the product, but it will be ignored.` not following this. Do you want to say that `https://install.stackrox.io/scanner/scanner-vuln-updates.zip for updating Scanner vulnerabilities in offline-mode` will continue to be populated? or Do you mean we will continue to populated the file `k8s-istio.zip` bundled in the image?

Reply:
we will continue to populate the `k8s-istio.zip` file found inside of `scanner-vuln-updates.zip` which is downloaded as part of the offline-dumps, but it is no longer needed (`scanner-vuln-updates.zip` is a ZIP of ZIPs)